### PR TITLE
Add documentation for content server API

### PR DESCRIPTION
**Pull request by agusaldasoro**
Created a MD file to store all the endpoints related to Content Server API.

The documentation is work in progress, only two endpoints are documented.

The template for the documentation is from: https://github.com/jamescooke/restapidocs

### CODE REVIEW

File: docs/content-server-api/deployments/get.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+  - Description: This value is used as the field to order all the deployments in the collection. If no parameter is sent, then the default field to order with will be local_timestamp.
+
+- sortingOrder
+  - Format: string
+  - Value: [ASC,DESC]
+  - Example: sortingOrder=ASC
+  - Default value: DESC
+  - Description: This value is used as the field to order all the deployments in the collection. If no parameter is sent, then the default order will be desc.
+
+- showAudit
```
Comment (nchamo) [inline docs/content-server-api/deployments/get.md:72]: Let's hide this. It was necessary for a migration, but we should even remove it from the code

File: docs/content-server-api/history/get.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  "pagination": {
+    "offset": 0,
+    "limit": 500,
+    "moreData": true
+  }
+}
+```
+
+
+## Notes
```
Comment (nchamo) [inline docs/content-server-api/history/get.md:73]: I think we can remove this, right?

File: docs/content-server-api/deployments/get.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+  ],
+  "filters": {},
+  "pagination": {
+    "offset": 0,
+    "limit": 500,
+    "moreData": true
+  }
+}
+
+## Notes
```
Comment (nchamo) [inline docs/content-server-api/deployments/get.md:385]: I think we can remove this, right?

File: docs/content-server-api/CONTENT_SERVER_API.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Content Server API
+
+## Endpoints
+
+* [Audit] (audit/:type/:entityId/get.md) : `GET /audit/:type/:entityId`
+* [Challenge] (challenge/get.md) : `GET /challenge`
+* [Contents per hashId] (contents/hashId/get.md) : `GET /contents/:hashId`
+* [Available Content] (contents/available-content/get.md) : `GET /available-content`
```
Comment (nchamo) [inline docs/content-server-api/CONTENT_SERVER_API.md:8]: Are these supposed to be links?

Comment (agusaldasoro) [reply docs/content-server-api/CONTENT_SERVER_API.md:8]: The idea of this schema is to have one file per method, in the future we should have all the endpoints+methods documented. Do you think that it's a good idea to create the files as empty and put "Work in progress" there?

Comment (nchamo) [reply docs/content-server-api/CONTENT_SERVER_API.md:8]: I see 2 options:
* Have links for all endpoints, and have a `Work in progress` like you said
* Have links for only the documented endpoints

I'm fine either way

**Review by nchamo — APPROVED** [review]
Great work!

**Pull request merged**